paper-server/patches/sources/net/minecraft/server/players/PlayerList.java.patch

```diff
@@ -557,7 +557,7 @@
 +        // Call post respawn event
 +        new com.destroystokyo.paper.event.player.PlayerPostRespawnEvent(
 +            serverPlayer.getBukkitEntity(),
-+            org.bukkit.craftbukkit.util.CraftLocation.toBukkit(vec3),
++            org.bukkit.craftbukkit.util.CraftLocation.toBukkit(serverPlayer.position(), serverPlayer.level(), serverPlayer.getYRot(), serverPlayer.getXRot()),
```

Comment:
Maybe can use the teleportTransition for all the data? in teory is the same but in the other places of this method is what is used rather than get the data from the serverPlayer